src/v/pandaproxy/schema_registry/README.md

```diff
@@ -0,0 +1,166 @@
+# Schema Registry
+
+## References
+
+- REST API: https://docs.confluent.io/platform/current/schema-registry/develop/api.html
```

Comment:
should we add refs to the aiven registry github

Reply:
Sure, done